modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -105,42 +107,133 @@ Ensure the two VPCs are using different network CIDRs, and after creating the VP
 ... Under *Destination*, enter the CIDR for the VPC in account B, which for this example is 172.20.0.0/16.
 
 ... Under *Target*, use the peer connection type point from the created peer connection.
+// this is where the changes for ROSA starts to line 396
+ifdef::openshift-rosa,openshift-rosa-hcp[]
+. Prepare the AWS account A IAM roles and policies:
+
+.. Create an IAM policy for the EFS CSI driver.
 
-. Create an IAM role, for example, “my-efs-acrossaccount-role” in AWS account B, which has a trust relationship with AWS account A, and add an inline AWS EFS policy with permissions to call “my-efs-acrossaccount-driver-policy”.
 +
-This role is used by the CSI driver's controller service running on the {product-title} cluster in AWS account A to determine the mount targets for your file system in AWS account B.
+[NOTE]
+====
+This has additional permission`s compared to a single account EFS CSI policy.
```

Comment:
that apostrophe looks out of place

Reply:
resolved, this is an OCP doc and must not have been captured before thanks